Source/iOS/UIColor+Hue.swift

```diff
@@ -173,6 +173,30 @@ public extension UIColor {
       return a
     }
   }
+
+	var hueComponent : CGFloat {
+		get {
+			var hue : CGFloat = 0
+			getHue(&hue, saturation: nil, brightness: nil, alpha: nil)
+			return hue
+		}
+	}
+
+	var saturationComponent : CGFloat {
```

Comment:
Can you use 2 spaces indentation?

Reply:
I think my Xcode defaults to 4 spaces indentation. I'll check.